crates/iota-rest-api/src/system.rs

```diff
@@ -121,7 +121,11 @@ pub struct SystemStateSummary {
     /// during safe mode.
     #[serde_as(as = "iota_types::iota_serde::BigInt<u64>")]
     #[schemars(with = "crate::_schemars::U64")]
-    pub safe_mode_computation_rewards: u64,
+    pub safe_mode_computation_charges: u64,
+    /// Amount of burned computation rewards accumulated during safe mode.
```

Comment:
```suggestion
    /// Amount of burned computation charges accumulated during safe mode.
```

Reply:
This suggestion will be outdated due to https://github.com/iotaledger/iota/pull/5270/commits/3d65d4a39724a007ab66be516b08e4bb0fc44c9d and https://github.com/iotaledger/iota/pull/5270/commits/3f5933339b60e7b381332c09e578516965fc801c

`IotaSystemStateSummaryV1` will now be kept unchanged with field `safe_mode_computation_rewards` and `IotaSystemStateSummaryV2` introduced with renamed and additional fields.

Reply:
Fixed with https://github.com/iotaledger/iota/pull/5276/commits/1ef7937537026cbeba44fa011f55458cdb1bc851